tests/test_aggregation.py

```diff
@@ -71,7 +81,9 @@ class AggeregateEngineTests(IsolatedAsyncioTestCase):
   def _equality_check(self, aidb_res, gt_res, error_target):
     assert len(aidb_res) == len(gt_res)
     for aidb_item, gt_item in zip(aidb_res, gt_res):
-      if abs(aidb_item - gt_item) / (gt_item) <= error_target:
+      relative_diff = abs(aidb_item - gt_item) / (gt_item)
+      print(f'% error: {relative_diff * 100}')
+      if relative_diff <= error_target:

```

Comment:
The logic is wrong. Why return True only one satisfies?

Reply:
The logic here is the same as before, just added variable called relative_diff

Reply:
Now you have several results, and you should return True all  satisfy,

Reply:
Changed, my bad